pkgs/applications/misc/calcoo/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, fetchzip
+, ant
+, jdk
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "calcoo";
+  version = "2.1.0";
+
+  src = fetchzip {
+    name = pname + "-" + version;
```

Comment:
```suggestion
```
Isn't this close to the default? If so we shouldn't repeat it.

Reply:
The default here is `source`, and it is confusing to keep track on the messages.